internal/controllers/device_plugin_reconciler.go

```diff
@@ -176,7 +176,7 @@ func (dprh *devicePluginReconcilerHelper) getModuleDevicePluginDaemonSets(ctx co
 	devicePluginsList := make([]appsv1.DaemonSet, 0, len(dsList.Items))
 	// remove the older version module loader daemonsets
 	for _, ds := range dsList.Items {
-		if ds.GetLabels()[constants.DaemonSetRole] != constants.ModuleLoaderRoleLabelValue {
+		if _, ok := ds.GetLabels()[constants.KernelLabel]; !ok {
```

Comment:
Why do we have that diff in the labels of the DS between u/s and d/s?

Reply:
Because we released upstream before we tested downstream. That is why now we wait for testing to be completed downstream before releasing upstream.
BTW this means that the 1.0 --> 1.1 upgrade was probably broken upstream.